pkg/appfile/appfile.go

```diff
@@ -212,14 +212,6 @@ func (af *Appfile) PrepareWorkflowAndPolicy(ctx context.Context) ([]*unstructure
 		}
 	}
 
-	af.WorkflowSteps, err = step.NewChainWorkflowStepGenerator(
-		&step.Deploy2EnvWorkflowStepGenerator{},
-		&step.ApplyComponentWorkflowStepGenerator{},
-	).Generate(af.app, af.WorkflowSteps)
-
-	if err != nil {
-		return nil, err
-	}
```

Comment:
why these code removed?

Reply:
They are moved to parser.go:115 so that external policy and referenced override component/trait policy can be counted in application revision.